tensorflow/stream_executor/dnn.h

```diff
@@ -667,8 +667,31 @@ class PoolingDescriptor {
   std::vector<int64> strides_;
 };
 
-typedef int64 AlgorithmType;
-constexpr AlgorithmType kDefaultAlgorithm = -1;
+class AlgorithmType {
+  public:
```

Comment:
"public:" and "private:" should be indented one space.

Recommend running clang-format over this whole patch, it will make other changes as well.

Reply:
Thanks. Fixed all clang-format issues.